stacks/jhipster-online/2.33.1/devfile.yaml

```diff
@@ -0,0 +1,133 @@
+schemaVersion: 2.2.0
+metadata:
+  name: jhipster-online
+  version: 2.33.1
+  description: "Stack with JHipster Online on Red Hat OpenShift Dev Spaces. 
+    This stack include:
+    - JHipster 8.8.0. for generate Spring Boot 3.4.1 projects.
+    - generator-jhipster-micronaut 3.6.0 for generate Micronaut 4.7.2 projects.
+    - generator-jhipster-quarkus 3.4.0 for generate Quarkus 3.11.1 projects.
+    - generator-jhipster-dotnetcore 4.2.0 for generate .Net 8.0 projects.
+    Watch the demo video:
+    - https://youtu.be/b7xbcTAGNIQ?si=snE57Th4S3gPv_Vn"    
+  displayName: JHipster Online
+  icon: https://raw.githubusercontent.com/redhat-developer-demos/jhipster-online/main/jhipster-icon.png
+  website: https://start.jhipster.tech
+  tags:
+    - Java
+    - JHipster
+    - Angular
+    - Spring
+    - Quarkus
+    - Micronaut
+  language: Java
+  projectType: springboot
+projects:
```

Comment:
@maximilianoPizarro Due to the difference of che using `projects` and odo v3 using `starterProjects` could this stack include this source as both?

In addition to your `projects` entry you would also include the following `starterProjects` entry:

```yaml
starterProjects:
  - name: jhipster-online
    description: Stack with the JHipster Online on DevSpaces Red Hat OpenShift
    git:
      checkoutFrom:
        revision: main
      remotes:
        origin: https://github.com/redhat-developer-demos/jhipster-online
```

@ibuziuk Does `starterProjects` add any conflicts with Che tools?

Reply:
Actually disregard this feedback, odo v3 testing does expect no starter projects as well.

FYI @maximilianoPizarro @ibuziuk